tests/test_client.py

```diff
@@ -573,21 +574,49 @@ class Model(BaseModel):
         assert isinstance(response, Model)
         assert response.foo == 2
 
-    def test_base_url_setter(self) -> None:
-        client = Turbopuffer(
-            base_url="https://example.com/from_init", api_key=api_key, _strict_response_validation=True
-        )
-        assert client.base_url == "https://example.com/from_init/"
-
-        client.base_url = "https://example.com/from_setter"  # type: ignore[assignment]
-
-        assert client.base_url == "https://example.com/from_setter/"
-
```

Comment:
This test is no longer relevant now that the base URL isn't settable after client construction.